policy-suggest-ecs-agent.yaml

```diff
@@ -0,0 +1,70 @@
+APIVersion: 1
+label: recipe:policy-suggest:ecs-agent
+data:
+  recipes:
+    - name: Create ecs-agent rule
+      description: Creates an ecs-agent rule to allow ecs-agent to talk to aws control plane.
+      label: recipe:policy-suggest:ecs-agent
+      metadata:
+        - "@aporeto:author=aporeto"
+      associatedTags:
+        - "aporeto:recipe:placement=policy-suggest"
+        - "aporeto:recipe:placement=ecs-rules"
```

Comment:
Are we making a new category for this? Otherwise we will need UI help here to do so.

Reply:
Yes, we need UI help here. I am going to create a separate ticket with the details for that.